PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -42,6 +39,13 @@
 #include "Common/DataModel/EventSelection.h"
 #include "Tools/KFparticle/KFUtilities.h"
 
+/// includes KFParticle
+#include "KFParticle.h"
+#include "KFParticleBase.h"
+#include "KFPTrack.h"
+#include "KFPVertex.h"
+#include "KFVertex.h"
```

Comment:
Why did you move the headers?

Reply:
Sorry, that's a mistake. I moved these headers back to their original positions. 
I just found that if I add
``` 
#ifndef HomogeneousField
#define HomogeneousField
#endif
  ```
to the front of these headers, this error is resolved
> Error while checking build/O2Physics/o2 for [4d023e4](https://github.com/AliceO2Group/O2Physics/commit/4d023e49d9a460166c2102a1db063b14af5e33b3) at 2024-07-30 19:24:
> 
> ```
> ## sw/BUILD/O2Physics-latest/log
> /sw/SOURCES/O2Physics/6327-slc7_x86-64/0/PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx:600:19: error: 'SetField' is not a member of 'KFParticle'
> /sw/SOURCES/O2Physics/6327-slc7_x86-64/0/PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx:576:72: error: unused parameter 'bcWithTimeStamps' [-Werror=unused-parameter]
> ninja: build stopped: subcommand failed.
> ```
> 
> Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/6327/4d023e49d9a460166c2102a1db063b14af5e33b3/build_O2Physics_o2/pretty.html).